src/svea_sensors/launch/navsat.launch

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0"?>
 <launch>
     <arg name="delay"       default="2.0" />
-    <arg name="yaw_offset"  default="0"/>
 
     <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform_node" respawn="true" output="screen">
         <param name="publish_filtered_gps" value="true"/>
         <param name="wait_for_datum" value="false"/>
         <param name="delay" value="$(arg delay)" />
         <param name="magnetic_declination_radians" value="0.1460841"/>
-        <param name="yaw_offset" value="$(arg yaw_offset)"/>
+        <param name="yaw_offset" value="1.5707963"/>
```

Comment:
This shouldn't be hard coded or...?

Reply:
The yaw_offset is based on the documentation of the IMU, saying the mapping for the cardinal directions. So, it should be hard coded. The parameter we should be changing during initialization is initial_pose_a, which define the starting yaw for the SVEA.